src/pages/InputNewTea.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import InputForm from '../components/InputForm'
+
+export default function InputNewTea({ setTea }) {
+  return (
+    <>
+      <InputForm setTea={setTea} />
```

Comment:
Siehe meinen Kommentar in der Form, vielleicht kannst du noch den Link Button in dieser Komponente hier einfügen und aus der Form herausziehen, damit in der Form wirklich nur die Sachen sind, die zur Form gehören. Sonst (so wie es jetzt ist) bräuchtest du nämlich diese Komponente hier tatsächlich auch garnicht 😊